github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -445,7 +445,7 @@ presubmits:
         - name: KUBEVIRT_STORAGE
           value: hpp
         - name: KUBEVIRT_MEMORY_SIZE
-          value: 9G
+          value: 10G
```

Comment:
I wonder if we should also increase the memory request. What do you think?

Reply:
Yeah happy to do that as well I just wasn't sure what the infra we are deploying to actually looks like, IOW if we had >44G to play with. I'll bump it now assuming we can.

Reply:
@dhiller keep me honest